### ISSUE TITLE
Indent rule regression after update from v3 to v4

### ISSUE THREAD

**Issue by paveleremin** [eslint/eslint#8846]
* **ESLint Version:**
"eslint": "^4.1.1",
"eslint-loader": "^1.8.0",
"eslint-plugin-angular": "^3.0.0",

* **Node Version:**
v8.1.2

* **npm Version:**
5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
```
"indent": ["error", 4, {"SwitchCase": 1}],
```

**What did you do? Please include the actual source code causing the issue.**
```js
import angular from 'angular';

module.exports = angular.module('ngMaterial', [
    'ng',
    require('angular-animate'),
    require('angular-aria'),

    require('./core'),
    require('./backdrop'),
    require('./button'),
    require('./checkbox'),
    require('./content'),
    require('./datepicker'),
    require('./dialog'),
    require('./divider'),
    require('./icon'),
    require('./input'),
    require('./menu'),
    require('./menuBar'),
    require('./panel'),
    require('./radioButton'),
    require('./showHide'),
    require('./sidenav'),
    require('./select'),
    require('./switch'),
    require('./tabs'),
    require('./toast'),
    require('./toolbar'),
    require('./tooltip'),
    require('./virtualRepeat')
])
.name; // <- 32 line
```

**What did you expect to happen?**
I expect no errors
**What actually happened? Please include the actual, raw output from ESLint.**
32:1  error  Expected indentation of 4 spaces but found 0  indent

**Comment by kaicataldo** [comment]
The indent rule was rewritten for v4 and is a bit more strict now. Chained method calls by default are expected to be indented once, however this can be overridden in your configuration. 

It sounds like you want this in your configuration:
```json
"indent": ["error", 4, {"SwitchCase": 1, "MemberExpression": 0}]
```

See the docs here: http://eslint.org/docs/rules/indent#memberexpression

**Comment by paveleremin** [comment]
Yes, I tried this option. Is it possible use both 0 and 1, i.e. "MemberExpression": [0,1]?

**Comment by kaicataldo** [comment]
@paveleremin It isn't, no. The rule is more strict now, with the tradeoff being that it's now autofixable!

**Comment by not-an-aardvark** [comment]
No, it's not possible to do that. However, you can do `MemberExpression: "off"` to ignore `MemberExpression` nodes completely.

**Comment by paveleremin** [comment]
And what about StandardJS they allow two variants?

**Comment by kaicataldo** [comment]
I believe StandardJS is still using ESLint 3.x.

**Comment by Aladdin-ADD** [comment]
yes. it did. https://github.com/standard/standard/pull/921#issuecomment-310211451

**Comment by not-an-aardvark** [comment]
Closing because this is working as intended.
